_data/picture.yml

```diff
@@ -20,11 +20,13 @@ presets:
     size: 100vw
 
   hero-card:
+    formats: [webp, original]
```

Comment:
Images using the none default format weren't serving a webp, increasing load. Added here.

Reply:
Good call.